content/guides/accessibility-for-teams/product-management.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2019-07-31 09:00:00 -0500
+title: "Accessibility for product managers"
+deck: ""
+summary: ""
```

Comment:
We have the option of setting a summary field for each guide page.

<img width="1054" alt="Screen Shot 2024-02-09 at 9 37 54 AM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/df50c28c-69ca-416f-a588-671f2486059e">

Reply:
`Summary` in the front matter is equivalent to the head tag's required `meta description`; it's displayed in search engine results and social media shares. Should not be seen on any pages.

`Deck` is the optional section immediately below a page's title. A "deck" is a newspaper/journalism term; a smaller headline (a subtitle or 1-2 sentence blurb) below the main headline (title).

Am leaving out for now so code stuff can be addressed separately.